Dockerfile

```diff
@@ -19,7 +19,21 @@ EXPOSE $PUMA_PORT
 #
 # RUN chown $APPUSER:$APPUSER ./db/schema.rb
 
+RUN curl -sL https://deb.nodesource.com/setup_13.x | bash -
+RUN apt-get install -y nodejs
+
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+RUN apt-get update && apt-get install yarn
+
+RUN yarn install
+# Have to set SECRET_KEY_BASE here to arbitrary string, otherwise task doesn't run
+RUN SECRET_KEY_BASE=valuenotactuallyused rails assets:precompile
+
 ENV APPUID 1000
+
+RUN chown -R $APPUID:$APPUID /usr/src/app/tmp/cache
+
```

Comment:
Hi @starswan, could you refactor this to match the approach in https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/283/files, in particular

```
RUN curl -sL https://deb.nodesource.com/setup_13.x | bash - \
   && curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
   && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
   && apt-get update \
   && apt-get install -y nodejs yarn \
   && apt-get clean
```

Using one `RUN` for the install (with a clean at the end) reduces image size because we don't end up with cache layers containing APT cache entries.

It looks like the other PR failed because it was pulling 10.x rather than 13.x.